File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/TeamRunnerInfoResponse.java
package lunalauf.rms.utilities.network.communication.message.response;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.ResponseType;

public class TeamRunnerInfoResponse extends RunnerInfoResponse {

    @SerializedName("team")
    public String teamName;
    @SerializedName("team-rounds")
    public int numTeamRounds;
    @SerializedName("minigame")
    public int teamMinigamePoints;

    public TeamRunnerInfoResponse() {
        super(ResponseType.TEAMRUNNER_INFO);
    }

}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/TeamRunnerInfoResponse.kt
package lunalauf.rms.utilities.network.communication.message.response

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.ResponseType

class TeamRunnerInfoResponse : RunnerInfoResponse(ResponseType.TEAMRUNNER_INFO) {
    @SerializedName("team")
    var teamName: String = "-"

    @SerializedName("team-rounds")
    var numTeamRounds = 0

    @SerializedName("funfactors")
    var teamFunfactorPoints = 0
}
